src/View/BakeView.php

```diff
@@ -187,12 +187,11 @@ protected function _evaluate($viewFile, $dataForView)
     /**
      * Get the contents of the template file
      *
-     * @param string $name A template name or a Bake template string
+     * @param string $filename A template filename
      * @return string Bake template to evaluate
      */
-    protected function _getViewFileContents($name)
+    protected function _getViewFileContents($filename)
     {
-        $filename = $this->_getViewFileName($name);
```

Comment:
If I ever git-bisect to this commit I will have no real idea why this commit was made.

Was `_getViewFileName` returning the passed input when it's a filepath, except in some circumstances or something? I don't pretend the previous code was right - but knowing in what way the previous code caused a problem would help understand "the problem".